From 1d94aa53274a78243d2dae9c5dd285cd1116d589 Mon Sep 17 00:00:00 2001
From: Osamu Aoki <osamu@debian.org>
Date: Sun, 10 May 2015 09:18:21 +0900
Subject: [PATCH 2/2] Use POSIX compliant find syntax

expr1 -or expr2
       Same as expr1 -o expr2, but not POSIX compliant.
---
 dh_compress | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/dh_compress b/dh_compress
index 0716fdd..ee51322 100755
--- a/dh_compress
+++ b/dh_compress
@@ -112,8 +112,8 @@ foreach my $package (@{$dh{DOPACKAGES}}) {
 				2>/dev/null || true;
 			find usr/share/doc \\
 				\\( -type d -name _sources -prune -false \\) -o \\
-				-type f \\( -size +4k -or -name "changelog*" -or -name "NEWS*" \\) \\
-				\\( -name changelog.html -or ! -iname "*.htm*" \\) \\
+				-type f \\( -size +4k -o -name "changelog*" -o -name "NEWS*" \\) \\
+				\\( -name changelog.html -o ! -iname "*.htm*" \\) \\
 				! -iname "*.xhtml" \\
 				! -iname "*.gif" ! -iname "*.png" ! -iname "*.jpg" \\
 				! -iname "*.jpeg" ! -iname "*.gz" ! -iname "*.taz" \\
-- 
2.1.4

